provision/acc_provision/acc_provision.py

```diff
@@ -1518,6 +1520,8 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
             network_operator_platform_CR = network_operator_platform_CR_template.render(config=netopConfig)
             network_operator_CR_output = network_operator_CR_template.render(config=netopConfig)
             network_operator_yaml = network_operator_spec_output + "\n---\n" + network_operator_CR_output + "\n---\n" + network_operator_platform_CR
+            network_operator_openshift_yaml = network_operator_openshift_spec_output + "\n---\n" + network_operator_CR_output + \
+                "\n---\n" + network_operator_platform_CR
```

Comment:
How will this file get written? I see that in L1541 below we are only writing network_operator_yaml.

Reply:
Good catch, fixed it!